quupe-RN/js/screens/SignIn/SignInContainer.js

```diff
@@ -1,14 +1,57 @@
 import React, { Component } from 'react';
+import { Mutation } from 'react-apollo';
+import { View, Text } from 'react-native';
+import { connect } from 'react-redux';
+import gql from 'graphql-tag';
 import SignIn from './SignIn';
+import { createUserToken } from '../../redux/modules/Token';
+import realm, { createToken } from '../../config/models';
+
+import styles from './styles';
+
+const signInMutation = gql`
+  mutation authenticateUser(
+    $email: String!
+    $password: String!
+  ) {
+    authenticateUser(
+      email: $email
+      password: $password
+    ) {
+      id
+      token
+    }
+  }
+`;
 
 class SignInContainer extends Component {
+  realmToken(token, id) {
```

Comment:
Please change this function name to saveAuthToken and inside this function you can put a little comment saying saving token to realm